compiler/resolution/preFold.cpp

```diff
@@ -1915,7 +1913,7 @@ static Expr* dropUnnecessaryCast(CallExpr* call) {
   return result;
 }
 
-static SymExpr* symOrParamExpr(Symbol* arg) {
+SymExpr* symOrParamExpr(Symbol* arg) {
```

Comment:
If this method is to be not just file-local (static) I think it needs a better name. One reasonable name might be something like `createTagActual` and another might be `createSymExprPropagatingParam`.